Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/ProjectFileHandler.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Amazon.Lambda.Annotations.SourceGenerator.FileIO;
+using System;
+using System.Xml;
+
+namespace Amazon.Lambda.Annotations.SourceGenerator
+{
+    /// <summary>
+    /// Utilities for working with project (.csproj) files, such as allowing projects
```

Comment:
nit: I'd exclude "such as allowing projects to opt out of the CloudFormation description modification." from the class description as it is implying the class is more specific than it's supposed to be.

Reply:
Fixed in amended commit.